src/script/backup/CrossPlatformBackup/CPB.export.ts

```diff
@@ -63,62 +68,55 @@ export const exportCPBHistoryFromDatabase = async ({
   }
 
   // Taking care of conversations
-  const {
```

Comment:
Whats new is, that the zod check checks now every single entry instead of the full array, with added error handling. 

This prevents us from throwing away ALL entries if a single entry doesnt fullfill the type check.

Same change is done for users and events